README.md

```diff
@@ -96,19 +96,19 @@ Before running or contribute to this project, you need to have the setup of the
 ## Test link deployment
 ### Deploy your changes on Vercel (Recommended Option)
 
-1. Commit, push your changes, and create pull request. Vercel will automatically deploy your application.
+1. Commit, push your changes, and create a pull request. GitHub Actions will automatically deploy your application and generate a test link.
 
-2.  Register your application [here](https://developers.binary.com/applications/). This will give you the ability to redirect back to your deployed page after login.
-    Use `https://YOUR_VERCEL_LINK/binary-static/en/logged_inws.html` for the Redirect URL and `https://YOUR_VERCEL_LINK/binary-static/en/redirect.html` for the Verification URL.
-    - Have api key ready from deriv app account in order to register
-    - Make note of your App ID to update in redmine card
+2.  Register your application [here](https://api.deriv.com/dashboard/). This will give you the ability to redirect back to your deployed page after login.
+    Use `<GENERATED_TEST_LINK_URL>/en/logged_inws.html` for the Redirect URL and `<GENERATED_TEST_LINK_URL>/en/redirect.html` for the Verification URL.
+    - Have your API token ready from app.deriv.com account in order to register
+    - Make note of your App ID in order to add it to your ClickUp card
```

Comment:
I'm not sure why we had this redmine mention here, since this shouldn't make sense to open source contributors. I'd recommend removing this line completely.